revenuecatui/src/iosMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/PaywallOptionsKtx.kt

```diff
@@ -69,7 +67,6 @@ internal class IosPaywallDelegate(
     }
 
     @Suppress("CONFLICTING_OVERLOADS", "PARAMETER_NAME_CHANGED_ON_OVERRIDE")
-    @ObjCSignatureOverride
```

Comment:
Ah yes, it's the suppression of course! It fails to compile without it. We would need some changes to PHC to avoid this problem. I added [SDK-3507](https://linear.app/revenuecat/issue/SDK-3507/add-a-rcpaywallviewcontrollerdelegate-to-phc-avoiding-conflicting).